packages/ui/src/models/visualization/flows/camel-rest-visual-entity.ts

```diff
@@ -148,6 +147,6 @@ export class CamelRestVisualEntity extends AbstractCamelVisualEntity<{ rest: Res
   }
 
   protected getRootUri(): string | undefined {
-    return this.rest.rest;

```

Comment:
`getRootUri` should return either `string` or `undefined`, here we were returning `Rest`